images/macos/provision/core/nvm.sh

```diff
@@ -11,7 +11,7 @@ curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/$VERSION/install.sh | bash
 if [ $? -eq 0 ]; then
         . ~/.bashrc
         nvm --version
-        nodeVersions=("v10" "v12" "v14")
+        nodeVersions=$(get_toolset_value '.node.nvm_versions[]')
         for version in ${nodeVersions[@]}
         do
                 nvm install $version
```

Comment:
Maybe something like this and get rid of `v` prefix in the toolset?
```suggestion
                nvm install v${version}
```